janitor/functions.py

```diff
@@ -4154,10 +4213,14 @@ def process_text(
 
     :param df: A pandas dataframe.
     :param column: String column to be operated on.
-    :param args, kwargs: Arguments for parameters.
+    :param string_function: Pandas string method to be applied.
+    :param args: Arguments for parameters.
+    :param kwargs: Keyword arguments for parameters.
     :returns: A pandas dataframe with modified column.
-    :raises: KeyError if ``string_function`` is not a Pandas string method.
-    :raises: TypeError if wrong ``arg`` or ``kwarg`` is supplied.
+    :raises KeyError: if ``string_function`` is not a Pandas string method.
+    :raises TypeError: if wrong ``arg`` or ``kwarg`` is supplied.
+
+    .. # noqa: DAR402
```

Comment:
This allows ``darglint`` to leave out checking for specific errors. The ``..`` prefix ensures that the ``noqa: DAR402`` doesn't appear in the docs.

Here, a DAR402 is raised (_DAR402_: The docstring describes an exception not explicitly raised.) since the function _could_ raise a TypeError, but doesn't explicitly have one in the source code. We use this to ignore the ``darglint`` raised error.